obs-packaging/wait-obs.sh

```diff
@@ -92,22 +106,25 @@ EOT
 }
 
 main() {
-	run_in_docker  $@
+	run_in_docker $@
 	local no_wait_publish="false"
 	case "${1:-}" in
-		"-h"|"--help")
-			usage "Help" 0
-			;;
-		--no-wait-publish)
-			no_wait_publish="true"
-			shift
-			;;
-		-*)
-			usage "Invalid option: ${1:-}" 1
-			;;
+	"-h" | "--help")
+		usage "Help" 0
+		;;
+	--no-wait-publish)
+		no_wait_publish="true"
+		shift
+		;;
+	-*)
+		usage "Invalid option: ${1:-}" 1
```

Comment:
Is the trailing `1` there for a purpose?

Reply:
Yeah  usage takes that value to exit with that exit code.